### ISSUE TITLE
Improving the usability of playback speed control

### ISSUE THREAD

**Issue by mongolsteppe** [TeamNewPipe/NewPipe#7466]
### Checklist
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### What feature do you want?
Currently the playback speed menu blocks a significant amount of the screen/video playing i.e. 
![a (Medium)](https://user-images.githubusercontent.com/75075420/143924683-94920628-d7f3-4785-9107-7f0880e3a41e.jpg)
![b (Medium)](https://user-images.githubusercontent.com/75075420/143924689-d89408e7-64b7-4355-a88d-0f9bf4baa1a7.jpg)

I suggest adding an intermediate step, where when you tap on the playback speed button, an overlay shows up with a few preset speed steps (for quickly switching between playback speeds) and then three dots that would take you to the current 'advanced' menu, like so:
![d (Medium)](https://user-images.githubusercontent.com/75075420/143924712-f95da1c5-2509-4954-9243-a02cce5914f9.jpg)

#### Why do you want this feature?
I have a hunch the average user just wants to change the playback speed quickly (like on youtube natively), while probably changing the pitch is a bit more niche feature. Also, pressing the preset speed gets you where you want to go in one tap (as opposed to the current version where you have to change the increment you want and then tap + or -).

Thanks!

**Comment by litetex** [comment]
Was fixed with #7363.
Can be closed if no further objection occurs.

**Comment by mongolsteppe** [comment]
I don't think that issue covers this FR (which, if I understand correctly, has compacted the menu a bit, but the menu itself is still blocking the video). My FR is about creating an intermediate step as reflected in the 3rd screenshot (upper right part of the picture), that won't block the video, while it's playing.

**Comment by litetex** [comment]
Fair point.

However:
* This would also block the video partially
* I don't think that users open up the dialog that often - So I would classify the issue inside as a kind of "niche" feature

**Comment by arncor** [comment]
Hello,

I also think the speed control window is blocking he view of the video and as it's a function I'm using very often it's quite annoying for me.

Is there any plan for example to just add 2 buttons on the player :
- speed up +0,25x
- speed down -0,25x

Thanks
